### DISCUSSION TITLE
How do I forward a COM port with wfreerdp.exe?

### DISCUSSION THREAD

**Discussion by MarcelG4** [FreeRDP/FreeRDP#8118] [Q&A]
Hello, I would like the following Windows to Windows connection:

wfreerdp.exe /v:192.168.2.146 /port:3389 /u:user /p:password /cert-ignore /serial:COM3

This doesn't work.

**Comment by mfleisz** [comment]
Sorry but COM-Port redirection is currently not supported on Windows.

**Reply by MarcelG4** [reply]
Are there any plans to support it in the future?

**Reply by mfleisz** [reply]
Nothing yet so don't expect it to be available any time soon ...
